pipeline/Makefile

```diff
@@ -350,27 +353,27 @@ ecoli_ref.fastq.bz2.keep.assembly.fa: ecoli_ref.fastq.bz2.keep.abundfilt.keep
 	# 	cp ecoli.37/contigs.fa ecoli_ref.fastq.bz2.keep.assembly.fa
 
 orig_vs_ecoli_assembly.report: ecoliMG1655.fa ecoli_ref.fastq.bz2.keep.assembly.fa
-	dnadiff ecoliMG1655.fa ecoli_ref.fastq.bz2.keep.assembly.fa -p orig_vs_ecoli_assembly
+	dnadiff ecoliMG1655.fa ecoli_ref.fastq.bz2.keep.assembly.fa -prefix orig_vs_ecoli_assembly
 
 ecoli_dn.faa: ecoli_ref.fastq.bz2.keep.assembly.fa
-	/mnt/prokka-1.7/bin/prokka ecoli_ref.fastq.bz2.keep.assembly.fa --outdir ecoli_dn --prefix ecoli_dn --force
+	${HOME}/src/prokka/prokka-* ecoli_ref.fastq.bz2.keep.assembly.fa --outdir ecoli_dn --prefix ecoli_dn --force
```

Comment:
And according to https://github.com/ged-lab/2012-paper-diginorm/pull/6/files#diff-ba53c673353e6a7edb32d8c3bd0f6817R9 you shouldn't even have to use a full path, just call prokka